csharp/Sdk.Examples/Horizon/Tutorials/ProcessHistory/ProcessHistory.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Finbourne.Horizon.Sdk.Api;
+using Finbourne.Horizon.Sdk.Client;
+using Finbourne.Horizon.Sdk.Model;
+using NUnit.Framework;
+using Sdk.Examples.Horizon.Utilities;
+
+namespace Sdk.Examples.Horizon.Tutorials.ProcessHistory
+{
+    [TestFixture]
+    public class ProcessHistory : TutorialBase
+    {
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+
+        }
+
+        [OneTimeTearDown]
+        public void TearDown()
+        {
+
+        }
+
+        [Test]
+        public void Get_Latest_Runs()
+        {
+            var processInformation = ProcessHistoryApi.GetLatestRuns();
```

Comment:
make this an async method and add an `Assert.That` > 2

Reply:
I've had a read around but I'm still not 100% clear on how to implement this